src/js/Controllers/VehicleModemController.js

```diff
@@ -0,0 +1,229 @@
+import sdlController from './SDLController';
+
+class VehicleModemController {
```

Comment:
While this controller is only handling the PTU at the moment, I do think there are other things the vehicle modem could be used for later on.

Reply:
If the plan is to use this for other things besides the PTU in the future, maybe the member variables shouldn't be named so PTU specific. All three would be used for another feature if added to this class but all three are labeled `PTU...`

Reply:
I've separated out the RPCClient implementation in https://github.com/smartdevicelink/generic_hmi/pull/239/commits/2809eb2d74fd8d68ef46bd38fd52eaebfe76c91b. Created a SimpleRPCClient class similar to https://github.com/smartdevicelink/sdl_hmi/blob/develop/ffw/RPCSimpleClient.js

Reply:
Awesome, so now can we name this class `VehicleModemPTUController` or something PTU specific?

Also, I think Inheritance fits better than Composition in this case since the `VehicleModemController` doesn't contain anything besides the PTUClient.

Reply:
Well, the reason I separated out the SimpleRPCClient class from this class was to remove the PTU specific member variables. 

I think it doesn't need to be renamed anymore since now there is only one client variable and some public functions related to the PTU process. The class could be used for other Vehicle modem features in the future